app/tests/unit/viewer_request_lambda/test_diversion.py

```diff
@@ -0,0 +1,64 @@
+from src import viewer_request_lambda_function
+
+import json
+from mako.template import Template
+from unittest import mock
+
+def load_event(event_name, **data):
+    with open('./events/{}.json'.format(event_name)) as fd:
+        template_text = fd.read()
+
+    data['cookies'] = ';'.join(data.get('cookies',[]) + ['random=junk'])
+
+    event_text = Template(template_text).render(**data)
+    event = json.loads(event_text)
+    return event
```

Comment:
would be interesting to move the event generation code into late-python, like an event JSON factory

Reply:
Yeah, as long as we can unify everything under one coherent umbrella in terms of event contents, configurable fields, etc.